BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00319-03
\ No newline at end of file
+1.0.25-prerelease-00322-04
```

Comment:
Oops, my bad. I updated the version to 00321 with a previous PR not realizing you also touched the file here. You'll need to rebase this before merge.